board-games/test/board-game-test.js

```diff
@@ -0,0 +1,70 @@
+var assert = require('chai').assert;
+var Game = require('../src/board-game.js');
+
+describe('Board Game', function() {
+  it.skip('should have a name', function() {
+    var game = new Game({name: 'Checkers'});
+
+    assert.isFunction(Game);
+    assert.equal(game.name, 'Checkers');
+  });
+
+  it.skip('should have a category', function() {
+    var game = new Game({name: 'Chess', type: 'strategy'})
+
+    assert.equal(game.category, 'strategy');
+  });
+
+  it.skip('should have a complexity level', function() {
+    var game = new Game({name: 'Pandmeic', type: 'strategy', complexity: 7.6});
+
+    assert.equal(game.complexity, 7.6);
+  });
+
+  it.skip('should have a maximum number of players', function() {
+    var game = new Game({name: 'Ticket to Ride', type: 'family', maxPlayers: 5});
+
+    assert.equal(game.maxPlayers, 5);
+  });
+
+  it.skip('should have a minimum number of two players by default', function() {
+    var game = new Game({name: 'Backgammon', type: 'strategy'});
+
+    assert.equal(game.minPlayers, 2);
+  });
+
+  it.skip('should be able to have a different minimum number of players', function() {
+    var game = new Game({name: 'Catan', type: 'strategy', minPlayers: 3});
+
+    assert.equal(game.minPlayers, 3);
+  });
+
+  it.skip('should hvae a maximum nuumber of players', function() {
```

Comment:
Typo in 'have'!

Reply:
The ones before were minimum player count tests and this is a maximum player count test. I added a default test to set the default max at 2 as well, since many games are just 2 player games.